virtualization-formulas/host/metadata/form.yml

```diff
@@ -92,3 +92,34 @@ default_net:
     dhcp_end:
       $type: text
       $name: Last IP adress for DHCP
+
+tuning:
```

Comment:
Same as in the top comment. Missing `$name` if it is supposed to be translatable. But I see that more groups are missing it so I guess this can be part of different PR.

Reply:
I changed the ones I added here.